rootfs/tpls/.rtorrent.rc

```diff
@@ -58,3 +58,7 @@ schedule2 = monitor_diskspace, 15, 60, ((close_low_diskspace,1000M))
 method.insert = d.get_finished_dir, simple, "cat=$cfg.download_complete=,$d.custom1="
 method.insert = d.move_to_complete, simple, "d.directory.set=$argument.1=; execute=mkdir,-p,$argument.1=; execute=mv,-u,$argument.0=,$argument.1=; d.save_full_session="
 method.set_key = event.download.finished,move_complete,"d.move_to_complete=$d.data_path=,$d.get_finished_dir="
+
+# Improve session saving support to reduce disk usage and prevent loss of torrents
+schedule2 = session_save, 1200, 3600, ((session.save))
```

Comment:
We should also warn users that want to change/increase `3600` as stipulated in rtorrent wiki: https://github.com/rakshasa/rtorrent/wiki/Performance-Tuning#session-save

> Increase this interval, e.g. to 12 hours instead, if you think you're system is stable and/or you don't care about the possible loss of resume and local stat in the meantime